tests/OrgMode.hs

```diff
@@ -22,6 +22,14 @@ spec = do
   prop "prop_encodeLinearity" prop_encodeLinearity
   prop "prop_decodeLinearity" prop_decodeLinearity
 
+  -- Checking for regressions
+  context "+d f +d f" $ do
+    it "encode" $ prop_encodeLinearity (OrgMsg "+d f +d f")
+    it "decode" $ prop_decodeLinearity (Deadline "+d" (Time {date = Next Friday, tod = Nothing}))
```

Comment:
```suggestion
    it "decode" $ prop_decodeLinearity (Deadline "+d f" (Time {date = Next Friday, tod = Nothing}))
```
Otherwise seems unrelated to the context.

(But then we might want to check the "+d" alone as well?)

Reply:
Good point; I changed everything to `+d +d f` instead; we test for "someting else coming after the first `+d`" below anyways